release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,93 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-server-config {
+    description
+      "Configuration data for the netconf server/subsystem";
+
+    leaf netconf-transport-porotocol {
+      type enumeration {
+        enum SSH_v2 {
+          description
+            "Use SSH v2 only";
+        }
+        enum TCP {
+          description
+            "Use Transmission Control Protocol (TCP) only";
+        }
+        enum TLS {
```

Comment:
You are creating the capability for TLS yet no parameters relating to TLS (trust anchors, etc..)

Reply:
> You are creating the capability for TLS yet no parameters relating to TLS (trust anchors, etc..)

Can you please help in giving reference to adding  the parameters related to TLS . Could not find any in Openconfig

Reply:
We don't currently have TLS details in the models -- assumed to be set up separately -- using gNOI, for example.

Reply:
At a bare minimum, this would need to be similar to the `certificate-id` string under `/system/grpc-server` however this creates a mandate for gRPC/gNOI for certificate management if one solely wants to manage via NETCONF over TLS.

The modeling for alternate management plane protocols should ideally not have a dependency on others